google-cloud-pub-sub-grpc/src/main/mima-filters/2.0.0-M3.backwards.excludes/Akka-gRPC-upgrade

```diff
@@ -0,0 +1,11 @@
+ProblemFilters.exclude[Problem]("com.google.*")
```

Comment:
Maybe add comment why we don't care about bin compat.

Reply:
Thanks, I added it to the docs now.